src/altinn-app-frontend/src/utils/layout/hierarchy.ts

```diff
@@ -426,6 +426,21 @@ export class LayoutNode<NT extends NodeType = 'unresolved', Item extends AnyItem
     return parents;
   }
 
+  /**
+   * Returns a list of rowIndices, sorted by most senior parent's rowIndex first
+   * Useful for accessing the datamodel related to a nested component
```

Comment:
Not really sure if we want to encourage that.. :wink: That's why `dataModelBindings` have already had their indexes injected when in a node hierarchy - we should encourage use of that directly. I'm not really sure we need this function at all, since (as mentioned) the row index already exists as part of the `id` property (although that is a risky design choice in itself, I think).

Reply:
Agree that that particular design choice is risky, but I guess we will keep it for the time being